src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -737,6 +762,33 @@ void StridedSlice::StridedSliceCommonExecutor::exec(const std::vector<MemoryCPtr
     });
 }
 
+void StridedSlice::StridedSliceCommonExecutor::execSliceScatter(const std::vector<MemoryCPtr>& srcMemory, const std::vector<MemoryCPtr>& dstMemory) {
+    const uint8_t* srcData = srcMemory[0]->getDataAs<const uint8_t>();
+    const uint8_t* srcUpdates = srcMemory[1]->getDataAs<const uint8_t>();
+    uint8_t* dstData = dstMemory[0]->getDataAs<uint8_t>();
+    cpu_memcpy(dstData, srcData, srcMemory[0]->getSize());
+    if (srcMemory[1]->getSize() == 0) {
```

Comment:
1. The Node interface has dedicated `isExecutable` method for such kind of checks. Please update it instead. 
2. Is it possible case that `Updates` input is not empty, while ``start``, ``stop``, ``step`` and ``axes`` are?

Reply:
1. Not sure id isExecutable is correct place for such check since execution would still require to copy all the data from input to output. This scenario might happen if all slice params (start/step, etc) would create empty slice, so updates would be expected to contain 0 dimension and to be empty. 
Without this if I saw cases where nThreads/workAmount was still calculated. Can investigate it further to fix it at earlier stage if required.
2. Yes, in that case updates would be expected to have same shape as data. Added test suite to custom tests that includes mentioned scenario

Reply:
Ok, that makes sence. Thanks for the explanation.